packages/application-components/src/components/modal-pages/internals/modal-page.styles.ts

```diff
@@ -1,22 +1,41 @@
 import { css, type SerializedStyles } from '@emotion/react';
 import { customProperties } from '@commercetools-uikit/design-system';
+import type { TModalPageSize } from './modal-page';
 
 export const TRANSITION_DURATION = 200;
 
 type StyleProps = {
   zIndex?: number;
-  size?: 'small' | 'large';
+  size?: TModalPageSize;
+};
+
+const getContainerSize = (size?: TModalPageSize) => {
+  switch (size) {
+    case 'small':
+      return '600px !important';
+    case 'large':
+      return '900px !important';
+    case 'xlarge':
+      return '1200px !important';
+    default:
+      return '100%';
+  }
 };
 
 export const getContainerStyles = (_props: StyleProps): SerializedStyles => css`
   position: absolute;
   top: 0;
   right: 0;
   height: 100%;
-  width: ${_props.size === 'small' ? '600px !important' : '100%'};
+  width: ${getContainerSize(_props.size)};
   display: flex;
   flex-direction: column;
-  background-color: ${customProperties.colorSurface};
+  background: linear-gradient(
+    270deg,
+    rgba(0, 0, 0, 0.05) 0%,
+    rgba(238, 238, 238, 0) 100%
+  );
+  background-blend-mode: darken;
```

Comment:
These colors are in the Figma file provided by design team. I have applied these to the container which made the container a bit darker than the previous color which was much whiter. Now the container is dark but the header in the regular modal pages is still white which is looking pretty weird now

I'm assuming design team intended the dark colors of the container to be applied for the header as well?

Reply:
Maybe we can have a call about this.
I don't see this panel background colors nor the gradient in the designs, but I might be missing something.

Reply:
Apologizes that was my own confusion, just had a call with Filip and he explain to me that the dark colors he mentioned were for the box shadow and not for the actual background of the modal 😅  silly mistake from my side. all good here. These 2 comments were resolved by Filip.

Reply:
After speaking with Filip, he said we need a new box shadow value for this one and he will speak with Carlos regarding that, once we have a decision there then we can continue with this PR.